app/Draft.js

```diff
@@ -1,7 +1,7 @@
 import crypto from 'crypto';
 
-import DraftModel from './objects/DraftModel';
-import PlayerModel from './objects/PlayerModel';
+import DraftModel from './objects/DraftModel.js';
```

Comment:
Did you have any errors about omitting the `.js`? A proper babel configuration should prevent this from being needed.

Reply:
This might just be lack of knowledge on my part. I had to add the js to make it work. Also ha to make some changes from requires to imports and add a type module in the package.json in order to run. We can discuss it further if you wish me to change it.